Assets/GothicVR/Scripts/UI/OpenSettingsFilePicker.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+using SFB;
+using GVR.Manager.Settings;
+using GVR.Bootstrap;
+
+namespace GVR
+{
+    public class OpenSettingsFilePicker : MonoBehaviour
+    {
+        public static void OpenGothicInstallationFilePicker()
+        {
+            var paths = StandaloneFileBrowser.OpenFolderPanel("Select Gothic installation","", false);
+
+            if (paths == null || paths.Length == 0)
+                return;
+            
+            GameSettings gameSettings = SettingsManager.I.LoadGameSettings();
+            gameSettings.GothicIPath = paths[0];
+            SettingsManager.I.SaveGameSettings(gameSettings);
+            
+            if (SettingsManager.I.CheckIfGothic1InstallationExists())
+            {
+                PhoenixBootstrapper.I.installationFilePicker.SetActive(false);
```

Comment:
This property is renamed. I assume we need to disable both: The button and the message?
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/d287b4e1-5fad-415e-8447-18af8ecaa677)

Reply:
No. The basic idea behind the last little implementation is that the text always show. But the button is only visible in PCVR. This gives the PCVR user the possibility to configure the path via the file picker. An Android user only gets the information that the files are missing. We might wanna show a different more detailed message for Android.